### PR TITLE
Update docs with the pager usage part

### PR DESCRIPTION
**Pull request by vz10**
*Issue #, if available:*

*Description of changes:*
Add part about how to set up or disable output to pager

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: awscli/topics/config-vars.rst [hunk 1]
```diff
@@ -60,29 +60,31 @@ General Options
...
+-------------------- -------------- --------------------- --------------------- ----------------------------
+parameter_validation N/A            parameter_validation  N/A                   Toggles parameter validation
+-------------------- -------------- --------------------- --------------------- ----------------------------
+tcp_keepalive        N/A            tcp_keepalive         N/A                   Toggles TCP Keep-Alive
+-------------------- -------------- --------------------- --------------------- ----------------------------
+max_attempts         N/A            max_attempts          AWS_MAX_ATTEMPTS      Number of total requests
+-------------------- -------------- --------------------- --------------------- ----------------------------
+retry_mode           N/A            retry_mode            AWS_RETRY_MODE        Type of retries performed
+-------------------- -------------- --------------------- --------------------- ----------------------------
+N/A                  --no-cli-pager N/A                   N/A                   Disable output to pager
```
Comment (kyleknap) [inline awscli/topics/config-vars.rst:86]: For the columns, I think we should fill this out with the other configuration options so for the first column it would be `cli_pager`, the third `cli_pager`, and the fourth `AWS_PAGER`. Then for the description we can set it as `Redirect output to a pager`

File: awscli/topics/config-vars.rst [hunk 2]
```diff
@@ -476,6 +478,38 @@ To set encoding used for text files different from the locale, you can use
 with default encoding ``CP1252``, setting ``AWS_CLI_FILE_ENCODING=UTF-8`` would
 make CLI ignore locale encoding and open text files using ``UTF-8``.
 
+Pager
+-----
+
+AWS CLI uses pager for output data that don't fit on the screen.
+
+By default if on Linux/MacOS, use ``less`` as the pager on Windows
```
Comment (kyleknap) [inline awscli/topics/config-vars.rst:486]: Let's reword this to: "If on Linux/MacOS, `less` is used as the default pager. On Windows, the default is `more`."

File: awscli/topics/config-vars.rst [hunk 3]
```diff
@@ -476,6 +478,38 @@ To set encoding used for text files different from the locale, you can use
...
+by default use ``more``
+
+    You can override the default pager with the following configuration
```
Comment (kyleknap) [inline awscli/topics/config-vars.rst:489]: I think it may make sense to break some of these paragraphs into their own subsubsection. So for this one, we can label it `Configuring pager` instead of having to rely on block quotes. So:
```
Configuring pager
^^^^^^^^^^^^^^^^^^
You can override the default pager with the following configuration options. The order of precedence is:

* ``AWS_PAGER`` environment variable
* ``cli_pager`` shared config variable
* ``PAGER`` environment variable

```

File: awscli/topics/config-vars.rst [hunk 4]
```diff
@@ -476,6 +478,38 @@ To set encoding used for text files different from the locale, you can use
...
+    options. These are in order of precedence:
+    ``AWS_PAGER`` environment variable
+    ``cli_pager`` shared config variable
+    ``PAGER`` environment variable
+
+If you set any of the configuration options to an empty string
+(e.g. ``AWS_PAGER=""``) or use ``--no-cli-pager`` option in the command line the
```
Comment (kyleknap) [inline awscli/topics/config-vars.rst:496]: I think it would be good to show the example on how to set the pager off for a profile and also for all profiles here.

File: awscli/topics/config-vars.rst [hunk 5]
```diff
@@ -476,6 +478,38 @@ To set encoding used for text files different from the locale, you can use
...
+CLI will not send the output to a pager.
+
+    If ``LESS`` environment variable isn't set CLI set it to ``FRX`` (see "less" manual
```
Comment (kyleknap) [inline awscli/topics/config-vars.rst:499]: Similar to a previous comment, instead of using block quotes, let's make this a new subsubsection `Pager settings`

File: awscli/topics/config-vars.rst (hunk 1 above)
Comment (vz10) [reply awscli/topics/config-vars.rst:86]: I was thinking about it. But `--no-cli-pager` works slightly different than the rest it doesn't `Redirect output to a pager`

Comment (kyleknap) [reply awscli/topics/config-vars.rst:86]: How about `Redirect/Disable output to a pager` then? This table is used for more general configuration options so if the pager functionality is included in this table it should included the config variable and environment variable.

File: awscli/topics/config-vars.rst [hunk 6]
```diff
@@ -476,6 +478,65 @@ To set encoding used for text files different from the locale, you can use
 with default encoding ``CP1252``, setting ``AWS_CLI_FILE_ENCODING=UTF-8`` would
 make CLI ignore locale encoding and open text files using ``UTF-8``.
 
+Pager
+-----
+
+AWS CLI uses pager for output data that don't fit on the screen.
```
Comment (kdaily) [inline awscli/topics/config-vars.rst:484]:
```suggestion
The AWS CLI uses a pager for output that does not fit on the screen.
```

File: awscli/topics/config-vars.rst [hunk 7]
```diff
@@ -476,6 +478,65 @@ To set encoding used for text files different from the locale, you can use
...
+
+If on Linux/MacOS, ``less`` is used as the default pager. On Windows,
```
Comment (kdaily) [inline awscli/topics/config-vars.rst:486]:
```suggestion
On Linux/MacOS, ``less`` is used as the default pager. On Windows,
```

File: awscli/topics/config-vars.rst [hunk 8]
```diff
@@ -476,6 +478,65 @@ To set encoding used for text files different from the locale, you can use
...
+the default is ``more``
```
Comment (kdaily) [inline awscli/topics/config-vars.rst:487]:
```suggestion
the default is ``more``.
```

File: awscli/topics/config-vars.rst [hunk 9]
```diff
@@ -476,6 +478,65 @@ To set encoding used for text files different from the locale, you can use
...
+
+    ``AWS_PAGER`` environment variable
+
+    ``cli_pager`` shared config variable
+
+    ``PAGER`` environment variable
+
+If you set any of the configuration options to an empty string
+(e.g. ``AWS_PAGER=""``) or use ``--no-cli-pager`` option in the command line the
+CLI will not send the output to a pager.
```
Comment (kdaily) [inline awscli/topics/config-vars.rst:503]:
```suggestion
AWS CLI will not send the output to a pager.
```

File: awscli/topics/config-vars.rst [hunk 10]
```diff
@@ -476,6 +478,65 @@ To set encoding used for text files different from the locale, you can use
...
+
+Examples
+""""""""
+
+To disable pager for ``default`` profile:
```
Comment (kdaily) [inline awscli/topics/config-vars.rst:508]:
```suggestion
To disable the pager for ``default`` profile:
```

File: awscli/topics/config-vars.rst [hunk 11]
```diff
@@ -476,6 +478,65 @@ To set encoding used for text files different from the locale, you can use
...
+
+    aws configure set cli_pager "" --profile default
+
+To disable pager for all the profiles in current terminal session:
```
Comment (kdaily) [inline awscli/topics/config-vars.rst:512]:
```suggestion
To disable the pager for all profiles in the current terminal session:
```

File: awscli/topics/config-vars.rst [hunk 12]
```diff
@@ -476,6 +478,65 @@ To set encoding used for text files different from the locale, you can use
...
+
+    export AWS_PAGER="" - for Linux
+
+    set AWS_PAGER="" - for Windows cmd
+
+To disable pager for one command call:
```
Comment (kdaily) [inline awscli/topics/config-vars.rst:518]:
```suggestion
To disable the pager for one command call:
```

File: awscli/topics/config-vars.rst [hunk 13]
```diff
@@ -476,6 +478,65 @@ To set encoding used for text files different from the locale, you can use
...
+
+    aws <command> <sub-command> --no-cli-pager
+
+
+Pager settings
+^^^^^^^^^^^^^^
+
+If ``LESS`` environment variable isn't set CLI set it to ``FRX`` (see "less" manual
```
Comment (kdaily) [inline awscli/topics/config-vars.rst:526]:
```suggestion
If the ``LESS`` environment variable is not set the AWS CLI will set it to ``FRX`` (see "less" manual
```

File: awscli/topics/config-vars.rst [hunk 14]
```diff
@@ -476,6 +478,65 @@ To set encoding used for text files different from the locale, you can use
...
+page for more information about possible options https://man7.org/linux/man-pages/man1/less.1.html)
+in order to set the appropriate flags. If you set the ``LESS`` env var, we will
+not clobber it with ours (e.g. ``FRX``). Be aware that different shells can
+have different default values for ``LESS`` environment variable that can cause
```
Comment (kdaily) [inline awscli/topics/config-vars.rst:530]:
```suggestion
have different default values for the ``LESS`` environment variable that can cause
```

File: awscli/topics/config-vars.rst [hunk 15]
```diff
@@ -476,6 +478,65 @@ To set encoding used for text files different from the locale, you can use
...
+unexpected behavior of CLI output
```
Comment (kdaily) [inline awscli/topics/config-vars.rst:531]:
```suggestion
unexpected behavior of AWS CLI output.
```

File: awscli/topics/config-vars.rst [hunk 16]
```diff
@@ -476,6 +478,65 @@ To set encoding used for text files different from the locale, you can use
...
+
+You can also set flags when specifying the pager and those will combine
+with any env vars we set (e.g. ``AWS_PAGER="less -S"`` will essentially make it
+less ``-FRXS``). The combining flags behavior is builtin into less.
```
Comment (kdaily) [inline awscli/topics/config-vars.rst:535]:
```suggestion
less ``-FRXS``). The behavior of combining flags is a feature of ``less``.
```

File: awscli/topics/config-vars.rst [hunk 17]
```diff
@@ -476,6 +478,65 @@ To set encoding used for text files different from the locale, you can use
...
+
+If ``LESS`` environment variable isn't set CLI set it to ``FRX`` (see "less" manual
+page for more information about possible options https://man7.org/linux/man-pages/man1/less.1.html)
+in order to set the appropriate flags. If you set the ``LESS`` env var, we will
+not clobber it with ours (e.g. ``FRX``). Be aware that different shells can
+have different default values for ``LESS`` environment variable that can cause
+unexpected behavior of CLI output
+
+You can also set flags when specifying the pager and those will combine
+with any env vars we set (e.g. ``AWS_PAGER="less -S"`` will essentially make it
```
Comment (kdaily) [inline awscli/topics/config-vars.rst:534]:
```suggestion
with any environment variables we set (e.g. ``AWS_PAGER="less -S"`` will make it
```

File: awscli/topics/config-vars.rst [hunk 18]
```diff
@@ -476,6 +478,66 @@ To set encoding used for text files different from the locale, you can use
...
+^^^^^^^^^^^^^^^^^^
+
+You can override the default pager with the following configuration
+options. These are in order of precedence:
+
+    ``AWS_PAGER`` environment variable
+
+    ``cli_pager`` shared config variable
+
+    ``PAGER`` environment variable
```
Comment (kyleknap) [inline awscli/topics/config-vars.rst:499]: Can we use bullet points here for the listing precedence instead of a block quote? So:
```
* ``AWS_PAGER`` environment variable
* ``cli_pager`` shared config variable
* ``PAGER`` environment variable
```
The format looks weird as a block quote otherwise

File: awscli/topics/config-vars.rst [hunk 19]
```diff
@@ -476,6 +478,66 @@ To set encoding used for text files different from the locale, you can use
...
+
+If you set any of the configuration options to an empty string
+(e.g. ``AWS_PAGER=""``) or use ``--no-cli-pager`` option in the command line the
+AWS CLI will not send the output to a pager.
+
+Examples
+""""""""
+
+To disable the pager for ``default`` profile:
```
Comment (kyleknap) [inline awscli/topics/config-vars.rst:508]: We should make these examples use code blocks. You should be able to do that by using double colons (e.g. `::`) instead of a single colon for each example.
